packages/extension/src/background/index.ts

```diff
@@ -15,6 +27,95 @@ const cacheAmplitudeDeviceId = async ({
   }
 };
 
+const excludedCompanionOrigins = [
+  'https://twitter.com',
+  'https://www.google.com',
+  'chrome-extension://',
+];
```

Comment:
This is kind of a first draft of hard-coded domains we would never want to fire on.
We can extend this or make this work differently in the future.

The idea was to have sites that devs use a lot, but never are featured blacklisted from Querying the API

Reply:
We can add stackoverflow, gmail, google meet, google calendar. Don't know if it makes sense

Reply:
Following up on our call yesterday, I see that `excludedCompanionOrigins` is your own implementation and not something we define on the manifest level. So it's great to have this list and when needed we can just remove sites from this list if we suddenly want to support them